Pixel Game/Assets/Scripts/Screen Scripts/GuiScreenManager.cs

```diff
@@ -57,6 +66,8 @@ bool Open(GuiScreen screen)
                 tabs.SetActive(true);
             }
             playerAttack.enabled = false;
+            playerMovement.enabled = false;
+            playerRigidBody.simulated = false;
```

Comment:
I like this. But why do you need to set `simulated` to false? Isn't it enough to disable movement script?

Reply:
Only disabling movement script made the character slide forward until hitting an obstacle since his movement was gone but his speed when traveling was still there.
Therefor I set simulated to false for movement and to stop players animation movement inset movement to false aswell :)

Reply:
Ok, got it

Reply:
A future improvement could be to create a public `Disable` function in `Movement` which handles this.